target/consensource-api
# Add tests for JWT auth failure

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

- Add test for JWT auth failure route - throws an `Unauthorized` error instead of a generic `Not Found`
- Update `docker-compose` files for testing to allow users to execute tests against a Postgres without running tarpaulin
- Remove instances of `rest_api` and replace with `api`
- Remove instances of `cert-registry` and replace with `consensource`

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- Follow README updates to run tests with code coverage. Confirm that the test coverage is `41.96%`
